src/core_modules/capture-core/components/TeiSearch/epics/teiSearch.epics.js

```diff
@@ -43,6 +44,32 @@ const getContextQueryArgs = (programId: ?string, trackedEntityTypeId: string) =>
 
 const getPagingQueryArgs = (pageNumber: ?number = 1) => ({ page: pageNumber, pageSize: 5 });
 
+export const searchTeiByTETIdEpic = (
+    action$: InputObservable,
+    store: ReduxStore,
+    { absoluteApiPath, querySingleResource }: ApiUtils,
+) =>
+    action$.pipe(
+        ofType(actionTypes.SEARCH_TEI_BY_TET_ID),
```

Comment:
NIT: Threw me off a bit, can we rename to something like: SEARCH_TE_IN_TET_SCOPE